packages/ui/src/working-groups/components/OpeningsList/Opening/OpeningListItem.tsx

```diff
@@ -33,9 +33,7 @@ export const OpeningListItem = ({ opening, past, onClick }: OpeningListItemProps
       <ToggleableItemInfo>
         <ToggleableItemInfoTop>
           <Subscription title={`ID: ${opening.id}`}>ID: {opening.runtimeId}</Subscription>
-          {isInFuture(opening.expectedEnding) && (
-            <Subscription>Ends {relativeTime(opening.expectedEnding)}</Subscription>
-          )}
+          <Subscription>Duration: {relativeTime(opening.createdAtBlock.timestamp)}</Subscription>
```

Comment:
![image](https://user-images.githubusercontent.com/6571453/208646950-62dc80aa-2630-4e15-b21e-72a9ca0c72b5.png)

I'm not 100% sure but in my understanding we should still show the time left until the `opening.expectedEnding` here. In which case it would still be a countdown and it should still stop showing after the `expectedEnding` has passed. Please could you confirm @dmtrjsg ?

Reply:
@thesan 💯 agreed, let's change the label from "Duration" to "Ends in" and calculate the countdown, instead of time elapsed from creation. 

Sorry for this, as the designs seem to be incorrect in Figma. Thank you all for understanding :)

Reply:
Fixed